content/ledger-objects/check.md

```diff
@@ -0,0 +1,54 @@
+## Check
+[[Source]<br>](https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/impl/LedgerFormats.cpp#L158 "Source")
+
+_Requires the [Checks Amendment](reference-amendments.html#checks)._
+
+A `Check` object describes a check, a potential pull payment, waiting to be cashed. (The potential payment has already been approved by its sender.) Example `Check` object:
+
+```json
+{
+  "Account": "rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo",
+  "Destination": "rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy",
+  "DestinationNode": "0000000000000000",
+  "DestinationTag": 1,
+  "Expiration": 570113521,
+  "Flags": 0,
+  "InvoiceID": "46060241FABCF692D4D934BA2A6C4427CD4279083E38C77CBE642243E43BE291",
+  "LedgerEntryType": "Check",
+  "OwnerNode": "0000000000000000",
+  "PreviousTxnID": "5463C6E08862A1FAE5EDAC12D70ADB16546A1F674930521295BC082494B62924",
+  "PreviousTxnLgrSeq": 6,
+  "SendMax": "100000000",
+  "Sequence": 2,
+  "index": "49647F0D748DC3FE26BDACBC57F251AADEFFF391403EC9BF87C97F67E9977FB0"
+}
+```
+
+A `Check` object has the following fields:
+
+| Field               | JSON Type        | [Internal Type][] | Description     |
+|:--------------------|:-----------------|:------------------|:----------------|
+| `LedgerEntryType`   | String           | UInt16            | The value `0x0043`, mapped to the string `Check`, indicates that this object is a Check object. |
+| `Account`           | String           | Account           | The sender of the Check. Cashing the Check debits this address's balance. |
+| `Destination`       | String           | Account           | The intended recipient of the Check. Only this address can cash the Check, using a [CheckCash transaction][]. |
+| `Flags`             | Number           | UInt32            | No flags are defined for Checks, so this value is always `0`. |
```

Comment:
I like this descr of `Flags`. Do you think it would be useful to use it for `Flags` in `amendments.md` as well -- is it applicable? It is more verbose, but I think it provides good info.